tests/integration/src/test/java/org/apache/pulsar/tests/integration/functions/PulsarStateTest.java

```diff
@@ -480,6 +480,7 @@ private void queryState(String functionName, String key, int amount)
             "--key", key
         );
         assertTrue(result.getStdout().contains("\"numberValue\": " + amount));
+        assertTrue(result.getStdout().contains("\"version\": " + version));
```

Comment:
It would be better to parse the JSON and validate that it doesn't contain extra keys such as `"stringValue": "\u0000\u0000\u0000\u0000\u0000\u0000\u0000\n"` which was observed in the release candidate validation.

Reply:
added below:

```
        assertFalse(result.getStdout().contains("stringValue"));
        assertFalse(result.getStdout().contains("byteValue"));
```